py/autoevals/oai.py

```diff
@@ -26,45 +29,91 @@ def open_cache():
     return _CONN
 
 
+def log_openai_request(span, input_args, response, **kwargs):
```

Comment:
currently this is only ever called with `span=current_span`, so should we remove the argument?

Reply:
good call